test/elixir_google_scraper_web/controllers/keyword_controller_test.exs

```diff
@@ -31,21 +33,79 @@ defmodule ElixirGoogleScraperWeb.KeywordControllerTest do
       assert redirected_to(conn) == Routes.keyword_path(conn, :index)
       assert get_flash(conn, :info) == "Your CSV file has been uploaded successfully"
     end
+
+    test "renders error message when CVS file is empty", %{
+      conn: conn
+    } do
+      file = %Plug.Upload{content_type: "text/csv", path: "test/fixture/empty_keywords.csv"}
+      user = insert(:user)
+      params = %{:file => file}
+
+      conn =
+        conn
+        |> log_in_user(user)
+        |> post(Routes.keyword_path(conn, :create), params)
+
+      assert redirected_to(conn) == Routes.keyword_path(conn, :index)
+      assert get_flash(conn, :error) == "File can't be blank"
+    end
+  end
+
+  describe "GET show/2" do
```

Comment:
If we handle not found case in the controller, we need to add a test case here too 🙌